### ISSUE TITLE
hsfm-sigmaNN: Covariances are too narrow sometimes

### ISSUE THREAD

**Issue by alexpostnikov** [alexpostnikov/Backprop-MPDM_torch#20]
hsfm-sigmann Covariances are too narrow sometimes

**Comment by warenick** [comment]
normalisation of input of data for covariance prediction NN, did covariance much better, but we need also normalize training data set on learning stage
